client/my-sites/patterns/components/header/index.tsx

```diff
@@ -1,23 +1,180 @@
+			textOnly: true,
+		},
+	},
 };
 
-export const PatternsHeader = ( { description, title }: PatternsHeaderProps ) => {
+const CONTENT: Record<
+	Category[ 'name' ],
+	{ description: Substitution; header: Substitution; title: Substitution }
+> = {
+	default: {
+		title: translate( 'WordPress Patterns', OPTIONS.home.title ),
+		header: translate( "It's easier with patterns", OPTIONS.home.header ),
+		description: translate(
+			'Dive into hundreds of expertly designed, fully responsive layouts, and bring any kind of site to life, faster.',
+			OPTIONS.home.description
+		),
+	},
+	header: {
+		title: translate( 'WordPress Header Patterns', OPTIONS.category.title ),
+		header: translate( 'Get a head start with patterns', OPTIONS.category.header ),
+		description: translate(
+			"Use predefined and beautiful patterns to decorate the header of your site, and catch your visitors' attention.",
+			OPTIONS.category.description
+		),
+	},
+	footer: {
+		title: translate( 'WordPress Footer Patterns', OPTIONS.category.title ),
+		header: translate( 'Build a strong foundation with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Choose from our collection of patterns to add essential information and navigation to the bottom of your site.',
+			OPTIONS.category.description
+		),
+	},
+	about: {
+		title: translate( 'WordPress About Patterns', OPTIONS.category.title ),
+		header: translate( 'Tell your story with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Add one of these patterns to your site to convey your mission and values compellingly.',
+			OPTIONS.category.description
+		),
+	},
+	posts: {
+		title: translate( 'WordPress Blog Post Patterns', OPTIONS.category.title ),
+		header: translate( 'Get more readers with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Take advantage of this selection of patterns that make your writing pop and keep readers engaged.',
+			OPTIONS.category.description
+		),
+	},
+	contact: {
+		title: translate( 'WordPress Contact Patterns', OPTIONS.category.title ),
+		header: translate( 'Connect with your audience using patterns', OPTIONS.category.header ),
+		description: translate(
+			'Pick from this set of patterns to add a contact form and a map to your site.',
+			OPTIONS.category.description
+		),
+	},
+	events: {
+		title: translate( 'WordPress Events Patterns', OPTIONS.category.title ),
+		header: translate( 'Promote events with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Highlight upcoming events on your site with patterns that grab attention and inform.',
+			OPTIONS.category.description
+		),
+	},
+	gallery: {
+		title: translate( 'WordPress Gallery Patterns', OPTIONS.category.title ),
+		header: translate( 'Stunning galleries with patterns', OPTIONS.category.header ),
+		description: translate(
+			'Pick from our set of patterns to showcase your visuals with elegance.',
```

Comment:
```suggestion
			'Pick from our set of patterns to showcase images in galleries and slideshows.',
```
Keywords, precision.